flexget/plugins/filter/series.py

```diff
@@ -949,6 +949,11 @@ def settings_schema(self):
             'additionalProperties': False
         }
 
+    @property
+    def supported_ids(self):
```

Comment:
where would you put it? it's used by both `configure_series` and now `series_list`. I thought filter_base was a natural place to put it